src/System.Management.Automation/engine/Interop/Windows/CreateFile.cs

```diff
@@ -0,0 +1,98 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#nullable enable
+
+using System;
+using System.IO;
+using System.Management.Automation;
+using System.Runtime.InteropServices;
+
+using Microsoft.Win32.SafeHandles;
+
+internal static partial class Interop
+{
+    internal static unsafe partial class Windows
+    {
+        // dwDesiredAccess of CreateFile
+        [Flags]
+        internal enum FileDesiredAccess : uint
+        {
+            GenericZero = 0,
+            GenericRead = 0x80000000,
+            GenericWrite = 0x40000000,
+            GenericExecute = 0x20000000,
+            GenericAll = 0x10000000,
+        }
+
+        // dwFlagsAndAttributes
+        [Flags]
+        internal enum FileAttributes : uint
+        {
+            Readonly = 0x00000001,
+            Hidden = 0x00000002,
+            System = 0x00000004,
+            Archive = 0x00000020,
+            Encrypted = 0x00004000,
+            Write_Through = 0x80000000,
```

Comment:
If we are not going to use original naming convention, i.e. `FILE_FLAG_WRITE_THROUGH`, use C# convention.

Reply:
It is a copy from old code. Really we have no need all the values and I kept them "just in case". Later we could correct style to follow .Net doc names.

Reply:
Really, there is no need to follow the names in public .NET API.